accelerator/main.c

```diff
@@ -20,7 +20,7 @@ static void ta_stop(int signal) {
   }
 }
 
-void check_iri_connection(ta_core_t* const core) {
+void health_track(ta_core_t* const core) {
```

Comment:
I think `iri_health_track` is more clearer.

Reply:
This thread will work for redis as well, so I prefer to keep the current naming